munininfluxdb/commands/cron.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import print_function
```

Comment:
This is one of the subcommands of the main command. It is loaded in `munininfluxdb/main.py`. In this particular case, it contains the `cron` code which I extracted from the other files. This way, the cron installation and deinstallation can be run separately (as also mentioned in the readme file).